README.md

```diff
@@ -108,7 +108,7 @@ If you're using Visual Studio Code for development, it's recommended that you:
 
    > **If you get *warnings* about installing dependencies** you probably don't worry about these
 
-   > **If you get *errors* about installing dependencies** you may need to run `npm install --legacy-peer-deps` or `npm install --force` (instead of `npm install`)
+   > **If you get *errors* about installing dependencies** you may need to run `npm install --legacy-peer-deps` or `npm install --force` (instead of `npm install`). When using `npm install` with `--legacy-peer-deps` flag, run `npx expo install` first.
```

Comment:
Please can you edit this line slightly -- folks should always run `npx expo install` first when using either flag (not only `--legacy-peer-deps`)

Please can you also change `npm install` to `npx expo install` on lines 105 and 147, so it's what people always use instead.  (The latter command also calls `npm install` as part of what it does.)  

Folks may well be used to using `npm install` as a habit, so please can you also add something extra on line 105 like "You should always use this command instead of `npm install`" and then link them to here https://github.com/Scottish-Tech-Army/Volunteer-app/blob/main/APP_DEVELOPMENT.md#installing-new-packages for more info

On line 148 please can you include again the extra sentence you've added above